### DISCUSSION TITLE
Cannot send data to subscriber running on seperate computer when publisher is on mobile data

### DISCUSSION THREAD

**Discussion by Jaynard2** [eProsima/Fast-DDS#3474] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

Data published on the device running with mobile data is sent to all relevant subscribers.

### Current behavior

Data published by the device on mobile data is sent to the subscriber running on the Discovery Server, but not to any other subscribers. The data generated by a publisher on a normal network is likewise not received by the subscriber on the mobile network.

### Steps to reproduce

Setup one Fast-DDS device to use mobile data and a second to use Wi-Fi (I do not have access to the network configuration). These can be either Windows 10 or Ubuntu 20.04. Setup a Discovery Server on a public server running Ubuntu 22.04. Add a publisher to either of the first two devices and a subscriber on all three. Any data published will make it to the Discovery Server only.

### Fast DDS version/commit

[57bf90af71bab82f2c6b24870e49d44b349083b2](https://github.com/eProsima/Fast-DDS/tree/57bf90af71bab82f2c6b24870e49d44b349083b2)

### Platform/Architecture

Ubuntu Focal 20.04 amd64

### Transport layer

TCPv4

### Additional context

Using port 5001.
The code used to configure Fast-DDS can be [viewed here](https://github.com/HEEV/Fast-DDS-Communication-Manager).

**Comment by JesusPoderoso** [comment]
Hello @Jaynard2 , thank you for the report.

According to [Fast DDS Contributing policy](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues), and since your question is related to a configuration or deployment issue, it has been moved to the discussion section.
A tutorial for configuring discovery server using WAN TCP is already in process, so we will let you know once it is uploaded to Fast DDS documentation. Regardless, let's see if we can provide an example in this thread.
